qf_lib/containers/series/qf_series.py

```diff
@@ -29,8 +28,10 @@ class QFSeries(pd.Series, TimeIndexedContainer):
     """
 
     def __init__(self, data: object = None, index: object = None, dtype: object = None, name: object = None,
-                 copy: object = False, fastpath: object = False):
-        if is_empty_data(data) and dtype is None:
+                 copy: bool = False, fastpath: bool = False):
+
+        empty_sequence_object = isinstance(data, Sequence) and not isinstance(data, str) and len(data) < 1
```

Comment:
doesn't "isinstance(data, Sequence)" imply "not isinstance(data, str)" ?

Reply:
Actually, a string is an instance of Sequence